admin_guide/revhistory_admin_guide.adoc

```diff
@@ -6,6 +6,20 @@
 :experimental:
 
 // do-release: revhist-tables
+== Tue Jun 14 2016
+// tag::admin_guide_tue_jun_14_2016[]
+[cols="1,3",options="header"]
+|===
+
+|Affected Topic |Description of Change
+//Tue Jun 14 2016
+
+|link:../admin_guide/overcommit.html[Overcommitting]
+|Added the link:../admin_guide/overcommit.html#configuring-masters-for-overcommitment[Configuring Masters for Overcommitment] section about the `*ClusterResourceOverride*` admission controller.

```

Comment:
@ahardin-rh This topic isn't in the OSD Cluster Admin guide, so you can drop revhistory for admin_guide (and for the book in revhistory_full.